admin/lib/generators/solidus_admin/component/USAGE

```diff
@@ -0,0 +1,13 @@
+Description:
+
+    Creates a SolidusAdmin component.
+
+Example:
+
+    $ bin/rails generate solidus_admin:component my_namespace/my_module_name my_attribute
```

Comment:
What do you think if we use `Rails.application.module_parent` to automatically fetch `my_namespace`?

We should also add `solidus_admin` within (like in `my_namespace/solidus_admin/my_module_name`) so that the generated component is automatically registered by the container.

In summary, we could:

```
bin/rails generate solidus_admin:component my_module_name my_attribute
```

have to generate:

`solidus_admin/my_namespace/my_module_name/component.rb` and siblings.

What do you think?

Reply:
Maybe we can add an option for that, I'd like to still leave the door open for defining components in an arbitrary namespace (e.g. `my_namespace_admin`) as I had some bad experiences with prependers using the same schema and introducing the requirement of using leading colons every time we had to reference `::Spree` constants (Basically every reference to `Spree` was looked up as `MyNamespace::Spree` creating constant headaches).

I'm also not sure if we're able to discern within the generator when we're working within the engine and when it's the host app (in order to understand when we should avoid the namespace), but maybe there's a way.

@waiting-for-dev please let me know if this is something you think should be addressed now or we can evaluate later.